packages/components/src/styling/tests/chromatic/sharegate.chroma.jsx

```diff
@@ -0,0 +1,145 @@
+import {
+    BackgroundColorMapping,
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong